### PR TITLE
New page for viewing quest activity

### PR DESCRIPTION
**Pull request by alalonde**
**What features/fixes does this PR include?**

This solves @peth-yursick's main pain point with quests, though not in the way he suggested. This is a WIP, but it adds a new page `/quest/activity` which displays in a table all the quests you have created as well as any proposals for those quests:

image

I also made some minor UI improvements to some of the links in the quests pages.

See #1310 for the original proposal.

## Follow up Improvement Ideas

This is a WIP, again, which needs:
1. Either a link to this new page from `/quests` OR
  - Move the logic into a new dashboard block type "Created Quests". This would require finagling the UI since the dashboard columns are so narrow
2. Create a new dashboard block type "Completed quests" Which could be repurposed from the existing player page "Completed quests" block (which is poorly named as it only shows _accepted quests_). The only difference is that the dashboard "completed quests" should only show your own pending quest submissions. 

There are some assumptions here per requirements, and I am out of time for now, but we could certainly just ship this with just a link to the new "quest activity" page.

### CODE REVIEW

**Comment by peth-yursick** [comment]
😱😱😱
Thank you very much, I know I've said it many times already but this will come in *very* handy & allow me to add more quests on there 😄

**Comment by peth-yursick** [comment]
I played around with it for a bit. Def too narrow to have all 3 fields but maybe "applicant name" not necessary 🤷‍♂️
Made it look like the leaderboard widget:
image
If @davort approves, I'll update the original issue & put this design there. (unless you think that issue should be closed with this & a new one open?)

**Comment by davort** [comment]
> I played around with it for a bit. Def too narrow to have all 3 fields but maybe "applicant name" not necessary 🤷‍♂️ Made it look like the leaderboard widget: image If @davort approves, I'll update the original issue & put this design there. (unless you think that issue should be closed with this & a new one open?)

Great job with reusing the leaderboard design!
Can't say I immediately understand how this should work and what it should do—e.g. what do the icons mean and why don't we use letters/numbers instead—but it definitely fits in the overall look & feel. I'm happy to help with future iterations of this design 😊

**Comment by alalonde** [comment]
I took the path of least resistance and just added a link to this new "Quest Activity" page. This should be gtg

**Review by dan13ram — APPROVED** [review]
I like this. Can create a dashboard box in a separate PR imo

File: packages/web/components/Dashboard/config.ts [hunk 1]
```diff
@@ -40,6 +40,7 @@ export const ALL_BOXES = [
   BoxTypes.DASHBOARD_SEEDS_INFO,
   BoxTypes.DASHBOARD_CALENDER,
   BoxTypes.DASHBOARD_LEADERBOARD,
+  BoxTypes.DASHBOARD_QUESTS,
```
Comment (dan13ram) [inline packages/web/components/Dashboard/config.ts:43]: this is not needed for now can be commented ?

**Pull request merged**